src/Engine/Manager/PhysicsManager.cpp

```diff
@@ -231,6 +251,18 @@ void PhysicsManager::SetMass(Component::RigidBody* comp, float mass) {
         comp->SetMass(mass);
 }
 
+void PhysicsManager::MakeKinematic(Component::RigidBody* comp) {
+    comp->MakeKinematic();
+}
+
+void PhysicsManager::MakeDynamic(Component::RigidBody* comp) {
+    comp->MakeDynamic();
+}
+
+void PhysicsManager::ForceTransformSync(Component::RigidBody* comp) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'ForceTransformSync' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)